src/tests/__init__.py

```diff
@@ -0,0 +1,12 @@
+from typing import List
```

Comment:
Don't type annotate tests. It is to much work. if mypy is not happy with that we will tell mypy to not check the tests.